data/hai/hai reveal 4 middle.txt

```diff
@@ -1418,6 +1454,7 @@ mission "Hai Reveal [A11] Sound the Alarm"
 		has "Hai Reveal [A10] Demanding Satisfaction: done"
 	on offer
 		event "hai and human ships at wormhole"
+		event "Hai election has started"
```

Comment:
It looks as if Hai is used lowercase in events for HR, so maybe change this and other event use cases to be lowercase?